test/test_and_analyze.jl

```diff
@@ -118,7 +174,35 @@ function code_instance(f, args...)
     end
 end
 
-macro test_unrolled(args_expr, unrolled_expr, reference_expr, contents_info_str)
+macro benchmark(expression)
+    return quote
+        Base.Experimental.@force_compile
```

Comment:
I'm wondering if we should use this-- it puts the test suite at risk for breaking without notice on version updates.

Reply:
Looks like we didn't need the `@force_compile` here; I just pushed an update that removes it. As for the use of `@force_compile` for the compilation time measurements, we need it in order to get accurate results, and it won't break when we switch to 1.11 because the same code is still being used in [base/timing.jl](https://github.com/JuliaLang/julia/blob/master/base/timing.jl). Also, when we switch to 1.11, we'll be able to use the publicly exported quantity `(@timed <expr>).compile_time`. I can open an issue to remember to do that after this is merged in.